site2/docs/architecture-overview.md

```diff
@@ -0,0 +1,143 @@
+---
+
+id: concepts-architecture-overview
+
+title: Architecture overview
+
+sidebar_label: Concepts
+
+---
+
+The following overview describes the components that make up a Pulsar cluster, from general to specific.  
+
+### Instance
+
+***
+
+A Pulsar instance is composed of one or more Pulsar clusters. Clusters within an instance can [replicate](concepts-replication.md) data amongst themselves.
+
+### Cluster
+
+***
+
+![Pulsar architecture diagram](/assets/pulsar-system-architecture.svg)
+
+In a Pulsar cluster:
+
+* One or more **brokers** handles and load balances incoming messages from **producers**, dispatches **messages** to **consumers**, communicates with the Pulsar **configuration store** to handle various coordination tasks, stores messages in BookKeeper instances (aka **bookies**), relies on a cluster-specific ZooKeeper cluster for certain tasks, and more.
+
+* A BookKeeper cluster consisting of one or more bookies handles [persistent storage](#persistent-storage) of messages.
+
+* A ZooKeeper cluster specific to that cluster handles coordination tasks between Pulsar clusters.
+
+An instance-wide ZooKeeper cluster called the Configuration Store handles coordination tasks involving multiple clusters, for example [geo-replication](concepts-replication.md).
+
+For a guide to managing Pulsar clusters, see the [clusters](admin-api-clusters.md) guide.
+
+### Producer
+
+***
+
+A producer is a process that attaches to a topic and publishes messages to a Pulsar [broker](reference-terminology.md#broker). The Pulsar broker processes the messages.
+
+Refer to the [producer](concepts-producer.md) topic for more information.
+
+### Topic
+
+***
+
+![Topic](/assets/producer-topic-consumer.svg)
+
+As in other pub-sub systems, topics in Pulsar are named channels for transmitting messages from producers to consumers. Topic names are URLs that have a well-defined structure:
+
+```http
+
+{persistent|non-persistent}://tenant/namespace/topic
+
+```
+
+| Topic name component | Description |
+|:--------------------|:-----------|
+| persistent / non-persistent | This identifies the type of topic. Pulsar supports two kind of topics: [persistent](concepts-architecture-overview.md#persistent-storage) and [non-persistent](#non-persistent-topics). The default is persistent, so if you do not specify a type, the topic is persistent. With persistent topics, all messages are durably persisted on disks (if the broker is not standalone, messages are durably persisted on multiple disks), whereas data for non-persistent topics is not persisted to storage disks.

```

Comment:
for the link of "persistent", you use `(concepts-architecture-overview.md#persistent-storage)`, that's correct
 
but for the link of `non-persistent`, why do you use `non-persistent-topics` which does not exist in the current file?

can you check all links before submitting a PR and attach all screenshots of your changes in the PR description? That's a way to test that all changes can be rendered successfully

Reply:
The reference was incorrect. Changed to admin-api-non-persistent-topics.md.

Reply:
After further research, I discovered that the reference is recursive. I removed it. Local preview looks good.
![7DA2A764-B6ED-4734-9D40-59F2DB852DBB](https://user-images.githubusercontent.com/94193423/179251193-11089fb8-1171-4991-95e9-720bc76fdc55.jpeg)